### PR TITLE
Add the ability to list :

### PR DESCRIPTION
**Pull request by mathieutu**
- all the permissions that the user inherits from his roles,
 - all the user’s permissions.

### CODE REVIEW

**Comment by mathieutu** [comment]
I think Travis has a problem: tests passed on 7.0 and 7.1 but all other build have failed, but not only on my PR.

**Comment by freekmurze** [comment]
Thanks!

**Comment by freekmurze** [comment]
Tagged as `v1.11.0`

**Comment by mathieutu** [comment]
My pleasure! :D

File: src/Traits/HasRoles.php [hunk 1]
```diff
@@ -277,4 +277,27 @@ protected function getStoredRole($role)
 
         return $role;
     }
+
+    /**
+     * Return all the permissions the user have via his roles.
+     *
+     * @return Collection
+     */
+    protected function getPermissionsViaRolesAttribute()
```
Comment (milroyfraser) [inline src/Traits/HasRoles.php]: @freekmurze @mathieutu in master branch still this is not update. It's just a method not an accessors `getPermissionsViaRoles` . + `getAllPermissions`

Comment (freekmurze) [inline src/Traits/HasRoles.php]: It felt better to have these as regular function and not as accessors. These functions are mentioned in the readme.

**Pull request merged**